documentation/sequence-diagrams/new-recipe.md

```diff
@@ -0,0 +1,15 @@
+```mermaid
+sequenceDiagram
+    User->>Frontend (Browser): Go to website
```

Comment:
This one will probably need to be updated since we will move this into collections